app/models/server/models/Messages.js

```diff
@@ -1146,6 +1162,25 @@ export class Messages extends Base {
 
 		return this.findOne(query, { sort: { ts: 1 } });
 	}
+
+	findAllSlackImportedMessagesWithFilesToDownload() {
+		const query = {
+			'slackFile.url_private_download': {
```

Comment:
Would be possible to be generic here rather than tied to Slack? Like `_importFile` with a property `source: "slack"`?

Reply:
Is it possible to keep the camelcase here?